tasks/base/CentOS-8/install_dependencies.yml

```diff
@@ -14,7 +14,7 @@
     - replace:
         path: "{{ item.path }}"
         regexp: '#baseurl=http://mirror\.centos\.org'
-        replace: 'baseurl=http://vault.centos.org'
+        replace: 'baseurl=http://mirror.centos.org'
```

Comment:
Is this an intentional change for this PR ? Can you explain why it's necessary ?

Reply:
Sorry, somehow I managed to get my PRs glued together, fumbling about with the GUI. I'm more of a CLI kind of fellow. 

CentOS has changed their repo URLs back, so the previous fix is now a regression. the "vault.centos.org" URLs now are a "404 / not found" error.